centreon/tests/e2e/cypress/e2e/Local-authentication/01-local-authentication/index.ts

```diff
@@ -291,7 +293,7 @@ Then('user can not change password unless the minimum time has passed', () => {
     .find('#validForm input[name="submitC"]')
     .click();
 
-  cy.getRefreshDataOnIframe()
+  cy.wait(1000)
```

Comment:
oula ! we enter the danger zone here

Reply:
Ok, I take it for a technical debt to follow up then. We might want to get rid of some of these iFrames in the future...